src/gateio_new_coins_announcements_bot/main.py

```diff
@@ -372,10 +372,7 @@ def sell():
 
                                     # add sell order sold.json (handled better in session.json now)
 
-                                    # TODO: check if this makes sense for id
-                                    id = 0
-
-                                    id = f"{coin}_{id}"
+                                    id = f"{coin}_{sell.id}"
```

Comment:
I'm unable to actively test this change due to the gate.io api not having a test-net for spot orders.
According to the python [gateio library](https://github.com/gateio/gateapi-python) `create_order` returns an `Order` object with an id which was intended to be used as primary key (as seen in this [discussion](https://discord.com/channels/900656563805642753/900657265567223819/924144927917625394) on Discord)

@busyuqboy 
Can you confirm that this was your intention or otherwise provide the correct way to fix this?

Reply:
that looks fine to me